packages/pipeline-services/src/migration/utils/index.ts

```diff
@@ -0,0 +1,5244 @@
+/*
+ * Copyright 2018-2021 Elyra Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// mock palette created by copying palette content sent to migrate function
+// and merging it into a single json object
+
+export const mockPalette: any = {
```

Comment:
Maybe these huge constants should go in a separate file, it's really hard to read it here

Reply:
Oh nevermind haha didn't realize that was the entire file